src/lib/ChatRoomManager.ts

```diff
@@ -240,3 +260,15 @@ export const fetchRooms = async (
 
     return rooms
 }
+
+export const storeLocalMessages = (messages: any) => {
+    localStorage.setItem(LOCAL_MESSAGES_KEY, JSON.stringify(messages))
```

Comment:
For the record, we (me + @bearni95) discussed on side that it's a temporary solution. We're gonna respect localStorage capacity.